ts/features/itwallet/playgrounds/components/ItwL3Section.tsx

```diff
@@ -32,14 +23,7 @@ export const ItwL3Section = () => {
 
   return (
     <View>
-      <ListItemHeader label="IT Wallet (L3 locally)" />
-      <ListItemSwitch
-        label="Enable locally L3 wallet"
-        value={isL3LocallyEnabled}
-        onSwitchValueChange={() => {
-          dispatch(itwSetL3LocallyEnabled(!isL3LocallyEnabled));
-        }}
-      />
```

Comment:
We could add the ability to manually **disable** the L3 status by leaving this switch but with a `itwSetFiscalCodeWhitelisted(false)` action dispatch. It could be useful for testing purposes. What do you think?